README.md

```diff
@@ -371,6 +373,11 @@ It accepts the following options:
   the LTI service that must be configured in the A+ site beforehand.
 * `lti_resource_link_id`: LTI exercise key
 * `lti_context_id`: LTI course key
+* `lti_open_in_iframe`: Open the exercise in an iframe inside the A+ page instead of a new window.
+  Doesn't take any parameters.
+* `lti_aplus_get_and_post`:  The exercise uses the A+ protocol to connect to the service.
+  The LTI launch parameters are appended to the A+ protocol parameters. This does not work with standard LTI services.
+  Doesn't take any parameters.
```

Comment:
Convert "Doesn't take any parameters."  into "This option does not take any parameters."